src/test/codegen/vec-extend_from_array.rs

```diff
@@ -0,0 +1,24 @@
+// compile-flags: -O -C panic=abort
```

Comment:
If it passes locally, debug assertions are a good bet: https://github.com/rust-lang/rust/search?q=debug+assertions+get+in+the+way
```suggestion
// compile-flags: -O -C panic=abort
// ignore-debug: the debug assertions get in the way
```
(although most builders have debug assertions enabled, so then this test will only run during a try run / bors merge)